DoggyFriction/Services/DebtAggregator.cs

```diff
@@ -65,21 +65,7 @@ public void AddTransaction(string creditor, string debtor, DebtTransaction trans
                 debt.Transactions.Add(transaction.Inverse());
                 debt.Amount -= transaction.Amount;
```

Comment:
Now that I've extracted `Reverse`, it feels logical to extract these amount calculations to `Debt`, too. We could just `Sum` the transactions' amounts.